lib/services/show.ts

```diff
@@ -29,5 +29,25 @@ export const saveShow = async (
     throw new DatabaseError(error);
   }
 
+  await saveShowToImported(data[0].id, spotifyId);
+
+  return data;
+};
+
+const saveShowToImported = async (
+  showId: Tables<'show'>['id'],
+  spotifyId: Tables<'show'>['spotify_id'],
+) => {
+  const supabase = createSupabaseServiceClient();
+
+  const { data, error } = await supabase.from('imported_show').insert({
```

Comment:
I thought having an `upsert` instead of `insert` here. if we have race condition but on the other hand we call `saveShowToImported` function after we successfully save a show to our DB, I think this should be fine.

Reply:
yeah looks 👍